apps/nextjs-website/src/helpers/auth.helpers.ts

```diff
@@ -0,0 +1,4 @@
+export const passwordMatcher =
```

Comment:
Also the `passwordMatcher`, here the problem will be how can we be in sync with the password policy defined on Cognito?

Reply:
From what I saw cognito doesn't expose the password policy, I don't really like the duplication of this logic, but I don't see another method to do it

Reply:
Let's create an activity to improve this part using the password policy taken from cognito (e.g. as described [here](https://stackoverflow.com/a/48681452)).

Reply:
@MikeAtUqido when you've created the activity, comment this thread, so it will be linked to the activity

Reply:
Here's the activity: [DEV-1113](https://pagopa.atlassian.net/browse/DEV-1113)